lib/kafka/connection.rb

```diff
@@ -75,6 +105,18 @@ def close
     #
     # @return [Object] the response.
     def send_request(request)
+      # Immediately block on the asynchronous request.
+      send_async_request(request).call
```

Comment:
I really like how the sync request pattern is just the same as the async, with an immediate blocking call. Nice!